build/paket.references

```diff
@@ -0,0 +1 @@
+Fable.React.Types
```

Comment:
I don't think you need Fable.React.Types in the build project, right? It is only for publishing packages and other build automation stuff